### PR TITLE
Remove logging and progress bars to stdout when -quiet flag used

### PR DESCRIPTION
**Pull request by chrishajas**
Also, don't print progress bars when -verbose flag used or to log file

Author: Chris Hajas <chajas@pivotal.io>

### CODE REVIEW

File: restore/restore.go [hunk 1]
```diff
@@ -151,7 +151,7 @@ func restoreData(gucStatements []utils.StatementWithType) {
 
 	filteredMasterDataEntries := globalTOC.GetDataEntriesMatching(includeSchemas, includeTables)
 	totalTables := len(filteredMasterDataEntries)
-	dataProgressBar := utils.NewProgressBar(totalTables, "Tables restored: ", logger.GetVerbosity() == utils.LOGINFO)
+	dataProgressBar := utils.NewProgressBar(totalTables, "Tables restored: ", true)
```
Comment (khuddlefish) [inline restore/restore.go]: We introduced the showProgressBar argument to allow for some progress bars to show outside of INFO logging mode and others to hide. Now that they are all passing "true" I think we can remove that property.

File: utils/log.go [hunk 2]
```diff
@@ -214,6 +214,6 @@ func NewProgressBar(count int, prefix string, showProgressBar bool) *pb.Progress
 	progressBar.ShowTimeLeft = false
 	progressBar.SetMaxWidth(100)
 	progressBar.SetRefreshRate(time.Millisecond * 200)
-	progressBar.NotPrint = !(showProgressBar && count > 0)
+	progressBar.NotPrint = !(showProgressBar && count > 0 && logger.GetVerbosity() == LOGINFO)
```
Comment (khuddlefish) [inline utils/log.go]: I think this should be || not &&?

Comment (chrishajas) [reply utils/log.go]: We want to print the progress bar when tables are present and the verbosity is LOGINFO. The NotPrint property = !(print conditions statement).

Comment (jmcatamney) [reply utils/log.go]: Nope, it should be &&, it would be || if they weren't in parentheses; `!(count && info)` = `!count || !info`.

Comment (khuddlefish) [reply utils/log.go]: Ok sounds good. I wasn't totally sure either way :P

File: restore/restore.go (hunk 1 above)
Comment (khuddlefish) [reply restore/restore.go]: Actually, I think I want to keep this for the verbose solution I'm working on so nevermind. @chrishajas Can you push this soon so I can rebase on top for what I'm working on?

**Pull request merged**